src/app/content/components/__snapshots__/BookBanner.spec.tsx.snap

```diff
@@ -1,8 +1,84 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
-exports[`BookBanner renders \`null\` when passed a page that isn't in the book tree 1`] = `null`;
+exports[`BookBanner renders \`null\` when passed a page that isn't in the book tree 1`] = `
```

Comment:
this is correct, since we made it so it renders an empty state, but the title of the test should be changed